atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/ThriftQueryWeighers.java

```diff
@@ -91,14 +96,15 @@ public QueryWeight estimate() {
                 return ImmutableQueryWeight.builder()
                         .from(DEFAULT_ESTIMATED_WEIGHT)
                         .numBytes(weight.get())
+                        .numDistinctRows(numRows)
                         .build();
             }
 
             @Override
             public QueryWeight weigh(T result, long timeTakenNanos) {
                 return ImmutableQueryWeight.builder()
                         .from(estimate())
-                        .timeTakenNanos((int)timeTakenNanos)
+                        .timeTakenNanos((int) timeTakenNanos)
```

Comment:
See above, but timeTakenNanos should take a long or this will be broken for times over ~3 seconds (Integer.MAX_VALUE nanoseconds)